docs/tutorials/elasticdl_local.md

```diff
@@ -59,40 +57,50 @@ minikube start --vm-driver=hyperkit \
 eval $(minikube docker-env)
 ```
 
-The above command-line option `--mount-string` exposes the directory `./data` on
-the host to Minikube, so we can bind mount it into containers running on the
-local Kubernetes cluster.
+The command-line option `--mount-string` exposes the directory `./data` on the
+host to Minikube as `/data`, which, we can later bind mount into containers
+running on the Kubernetes cluster.
+
+The command `minikube docker-env` returns a set of Bash environment variable
+exports to configure your local environment to re-use the Docker daemon inside
+the Minikube instance.
 
-The following command is necessary to enable [RBAC](https://kubernetes.io/docs/reference/access-authn-authz/rbac/)
-of Kubernetes. At first, we need retrieves the source code to get the RBAC
-configuration YAML file.
+The following command is necessary to enable
+[RBAC](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) of
+Kubernetes.
 
 ```bash
-git clone https://github.com/sql-machine-learning/elasticdl.git
-cd elasticdl
-kubectl apply -f elasticdl/manifests/elasticdl-rbac.yaml
+kubectl apply -f \
```

Comment:
I have tried this command on my personal computer and get the following error message:
`The connection to the server raw.githubusercontent.com was refused - did you specify the right host or port?`

Reply:
@brightcoder01 I could run this command in my mac.

```
kubectl apply -f https://raw.githubusercontent.com/sql-machine-learning/elasticdl/develop/elasticdl/manifests/elasticdl-rbac.yaml
```

Maybe its network problem.

Reply:
@brightcoder01 I can run this command too. Did you try it during the period when GitHub's recent outage?